react/src/components/atoms/lists/OrderedList/style.scss

```diff
@@ -0,0 +1,2 @@
+@import 'global';
+@import '01-atoms/global/elements';
```

Comment:
seems like elements should be treated different, if the project import only the components that don't have global/elements it will not have basic body html styles. I would say for now keep it in index.scss and into the base